### ISSUE TITLE
Empty keys should not be translated

### ISSUE THREAD

**Issue by vascosilvaa** [aralroca/next-translate#956]
**What version of this package are you using?**
1.6.0

**What problem do you want to solve?**
When programming, we tend to create all the necessary keys while implementing a certain feature. Only after the development is finished, keys are filled by a translator role with the correct translations for all the necessary languages.
That leads to a lot of untranslated/empty strings keys placeholders in the translation files.

Example: 
`{
 'my-key': ''
}`

When a value of a key is an empty string, translate method should not consider that key as a valid translation, since this key will be translated to an empty string, resulting in a bug on the UI where no content will be displayed.

**What do you think is the correct solution to this problem?**
An optional parameter **allowEmptyStrings** - **true** as default.
If **allowEmptyStrings** parameter is marked as false, it should log an error when an empty string is attempted to be translated and return the namespace and key as result of the translation.

**Are you willing to submit a pull request to implement this change?**
yes

**Comment by aralroca** [comment]
@vascosilvaa this is already done, isn't it?

**Comment by vascosilvaa** [comment]
It is! Thanks 🙂
